app/src/main/res/layout/test_detail.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@codestaintin 
Should this file be in the code seeing that the activity_detail.xml is already present?